src/shared/data/communication.data.ts

```diff
@@ -0,0 +1,22 @@
+export const RS_DOCS_COMMUNICATION_LINK = 'https://docs.rs.school/#/rs-school-chats';
+
+export const discordLinks = {
+  javascript: 'https://discord.com/invite/QvEYg7EaQ4',
+  'javascript-en': 'https://discord.com/invite/uW5cCHR',
```

Comment:
Let's rename it in `camelCase` style

Reply:
I suppose, we don't want to refactor half of the logic for courses at this branch, so I will leave it as is. The logic for about component, training programs and even course names spins around exactly these names and in need of refactoring.